package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@edx/edx-proctoring",
   "//": "Be sure to update the version number in edx_proctoring/__init__.py",
   "//": "Note that the version format is slightly different than that of the Python version when using prereleases.",
-  "version": "2.0.4",
```

Comment:
This was out of sync with the Python package version, so I had to bump this up to `2.0.6` to match.